.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -28,7 +28,7 @@ Please do not remove items from the checklist
   - [ ] Place Helm package under gh-pages branch in `release` dir
   - [ ] Update `index.yaml` file under gh-pages branch in `release` dir:
     ```
-        > helm repo index . --url https://mellanox.github.io/network-operator/release
+        > helm repo index . --url https://mellanox.github.io/network-operator/release --merge ./index.yaml
```

Comment:
This would not work.

The process of updating index is a bit more complex if we want to preserve old entries. i will update